config-linux.md

```diff
@@ -126,6 +127,12 @@ Each entry has the following structure:
 
 The same `type`, `major` and `minor` SHOULD NOT be used for multiple devices.
 
+Containers MUST NOT access any device node that is not explicitly referenced in
```

Comment:
the wording seems too strong here.  Maybe we can say that using a device node not referenced in the **devices** array is runtime-specific and not portable?

Reply:
Sure. Any suggestion for the wording? Just changing MUST NOT to MAY NOT sufficient?

Reply:
@giuseppe Updated, please check.

Reply:
thanks, LGTM